src/components/application_manager/test/CMakeLists.txt

```diff
@@ -167,7 +167,7 @@ list(APPEND LIBRARIES
 create_test(am_hmi_capabilities_test "${AM_HMI_CAPABILITIES_SOURCES}" "${LIBRARIES}")
 create_test(am_application_test "${APPLICATION_SOURCES}" "${LIBRARIES}")
 # TODO(AKutsan) APPLINK-25547 Uncoment command tests when sdl will support multithreading build
```

Comment:
@wolfylambova22, please remove this comment, and squash this commit with previous.